samples/server/petstore/jaxrs/jersey1/pom.xml

```diff
@@ -1,209 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
@phstudy please recover this file.

Did you manually purge the folder before regenerating the samples? Just wondering how this could have happened.

Reply:
`pom.xml` was removed by this [line].(https://github.com/OpenAPITools/openapi-generator/blob/master/bin/java-petstore-jersey1.sh#L34)